src/pages/Thanks/ThanksPage.vue

```diff
@@ -445,6 +445,12 @@ export default {
 			return this.kivaCards.filter(card => card.kivaCardObject.deliveryType === 'print');
 		},
 		activeView() {
+			// Show the donation only view if the user has only donated and not lent
+			if (this.showDafThanks
+				|| (this.receipt && this.receipt?.totals?.itemTotal === this.receipt?.totals?.donationTotal)
```

Comment:
Won't this line make it so that donation-only transactions won't see the new TY page? We still want the user to see the new TY page if there is a receipt and the receipt has only non-monthly donations.